src/Features/Core/Portable/QuickInfo/IndentationHelper.cs

```diff
@@ -18,25 +19,29 @@ internal static class IndentationHelper
         /// This operation will potentially split spans that cover multiple lines into separate spans.
         /// </summary>
         /// <param name="text"></param>
-        /// <param name="spans">The initial set of spans to align.</param>
+        /// <param name="classifiedSpans">The initial set of spans to align.</param>
```

Comment:
would it be too much churn to do refactoring of Indentationhelper to share code with IprojectionBufferFactoryServiceExtensions? logic seems same...

I am fine doing refactoring in other PR though.

Reply:
Agreed.  Seems like we have a lot of these concepts scattered around.  Unifying woudl be good.  I'm also 100% ok with that happening later.

Reply:
created https://github.com/dotnet/roslyn/issues/28172 for this refactoring